content/en/service_catalog/customize/import_entries_backstage.md

```diff
@@ -43,6 +43,22 @@ Upon import, the following occurs:
 
 **Note**: The Service Catalog processes the entire YAML file as a whole. If any section of the YAML file does not have `kind:component` or `kind:system`, the entire `catalog-info.yaml file` is rejected. Schema version v3.0 is required to use kind:system and the `dependsOn` field.
 
+### Example YAML for `catalog-info.yaml`
```

Comment:
```suggestion
### Example YAML for catalog-info.yaml
```

the monospaced code text doesn't render in titles and sometimes causes weird translation issues